Plugins/BenchmarkTool/BenchmarkTool+Baselines.swift

```diff
@@ -182,10 +182,7 @@ extension BenchmarkTool {
                 var subDirectory = storagePath
                 subDirectory.append(file.lastComponent!)
                 if let directoryName = file.lastComponent {
```

Comment:
Hmm, the line above `file.lastComponent` was force-unwrapped so this check should always succeed, or?

Reply:
Yes, refactored both of these force unwraps properly now with proper `let` extraction instead, thanks.